### ISSUE TITLE
Add a function to invoke a `@scheduled` Lambda

### ISSUE THREAD

**Issue by lpsinger** [architect/architect#1446]
**Is your feature request related to a problem? Please describe.**
I want to write an end-to-end test of an Architect `@scheduled` function that I have written.

**Describe the solution you'd like**
A method in `@architect/functions` (or another appropriate architect project) that allows me to programmatically trigger a scheduled Lambda, similar to https://arc.codes/docs/en/guides/developer-experience/local-development#testing-%40events-and-%40queues.

**Describe alternatives you've considered**
Mocking DynamoDB myself instead of using arc.tables(). That's lots of extra work!

**Comment by ryanblock** [comment]
An `@architect/functions` `arc.scheduled` implementation would likely be useful for testing only, and would ultimately be contrived. `arc.events|queues` use actual SNS / SQS event buses (and related API / SDK calls) to fire events, whereas scheduled events are triggered by Cloudwatch Events / Eventbridge.

I'm thinking the appropriate solution for this would be to either add some kind of an endpoint or some exportable interface (or both) to https://github.com/architect/plugin-lambda-invoker/ – probably the latter makes more sense and would be easier to use. For example:

```js
let sandbox = require('@architect/sandbox')
let { invoke } = require('@architect/plugin-lambda-invoker')

await sandbox.start()
await invoke({ 
  pragma: 'scheduled',
  name: 'my-scheduled-event',
  payload: { ok: true },
})
await sandbox.end()
```

Thoughts?

**Comment by lpsinger** [comment]
> I'm thinking the appropriate solution for this would be to either add some kind of an endpoint or some exportable interface (or both) to https://github.com/architect/plugin-lambda-invoker/ – probably the latter makes more sense and would be easier to use.

I agree!

**Comment by ryanblock** [comment]
Just pushed 1.2.0 RC, which exposes the raw `invoke` from `sandbox.start`: https://arc.codes/docs/en/guides/plugins/sandbox#invoke-function

Please let me know if that works for you!

**Comment by lpsinger** [comment]
Version 1.2.0 of what?

**Comment by lpsinger** [comment]
Oh, I see, of plugin-lambda-invoker.

**Comment by lpsinger** [comment]
How do I use this from a unit test? It looks like this is designed to use inside a plugin.

```ts
import sandbox from '@architect/sandbox'

beforeAll(async () => {
  await sandbox.start()
})

afterAll(async () => {
  await sandbox.end()
})

describe('ADS archiver', () => {
  it('works with mock responses', async () => {
    // Where do I import invoke from?
    invoke({ pragma: 'scheduled', name: 'ads' })
  })
})
```

**Comment by ryanblock** [comment]
I actually wound up implementing it pretty much exactly as in the code example above: https://github.com/architect/architect/issues/1446#issuecomment-1699793985

**Comment by ryanblock** [comment]
Related: since the `sandbox.start` `invoke` method was originally authored to be used in a CLI context, its error handling is almost non-existent, see: https://github.com/architect/sandbox/blob/main/src/invoke-lambda/_plugin.js#L15-L29

So if your Lambda fails in that test above, you won't really be able to tell. We should fix that (separately).

**Comment by ryanblock** [comment]
Per the above comment, new related issue: https://github.com/architect/architect/issues/1447

**Comment by ryanblock** [comment]
Published production 1.2.0 of `@architect/plugin-lambda-invoker`!
